boards/sensoan/sgw2/Kconfig

```diff
@@ -0,0 +1,56 @@
+# SGW-2 board configuration
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+# Copyright (c) 2024 Sensoan Oy
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_SGW2_NRF9160 || BOARD_SGW2_NRF9160_NS
+
```

Comment:
BOARD_SGW2 needs to select GPIO (maybe PINCTRL too) if BOARD_SGW2_NRF9160 OR CONFIG_BOARD_SGW2_NRF9160_NS are selected

Reply:
It is not selected, it defaults to on - defaulting to on means someone can disable it, selecting it means it cannot be disabled. Case in point if you configure a project for this board, disable it using menuconfig then build, it will fail to build because of the .c file dealing with the reset, thus the board must specifically select it and prevent it from being unselected